src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -87,12 +94,24 @@ public Path saveBlobDataToTempWorkspace(InputStream mediaFile, String fileName)
 
     @Override
     public void handleKedaInvocationForMediaRequests() {
-        Optional<MediaRequestEntity> mediaRequest = mediaRequestService.retrieveMediaRequestForProcessing();
-
-        mediaRequest.ifPresentOrElse(
-            this::processAudioRequest,
-            () -> log.info("No open requests found for ATS to process.")
-        );
+        int counter = 1;
+        List<Integer> mediaRequestIdsProcessed = new ArrayList<>();
+        OffsetDateTime cutoffTime = currentTimeHelper.currentOffsetDateTime().plusMinutes(config.getLoopCutoffMinutes());
+        while (currentTimeHelper.currentOffsetDateTime().isBefore(cutoffTime)) {
+            if (counter++ > 200) {
```

Comment:
can we remove this magic number and introduce a descriptive constant?

Reply:
done